### PR TITLE
MPW-I testing setup of the temperature sensors array

### PR DESCRIPTION
**Pull request by msaligane**

### CODE REVIEW

**Comment by msaligane** [comment]
@QiruiZhang Do you mind reviewing this?

**Comment by QiruiZhang** [comment]
@msaligane I checked the ReadMe in this PR through this link provided in your modification: https://github.com/idea-fasoc/OpenFASOC/blob/7252190ebf6dd722e1c8e8ac48e2bf815e22b11e/tapeouts/MPWI/testsetup/README.md#section. And it seems the images are not showed. Probably the image paths in ReadMe need to be changed to their absolute paths including the hierachies of openFASoC, like '/OpenFASOC/tapeouts/MPWI/testsetup/img/xxx.png', or maybe just the relative paths './img/xxx.png'?

**Comment by QiruiZhang** [comment]
Everything else looks fine to me

**Comment by QiruiZhang** [comment]
And you may want to add a new line (<br />) before the pip installs in ReadMe section 2.2 to make sure each of them start with a new line. Sorry I wasn't aware of that when I modified that part.

**Comment by msaligane** [comment]
Thanks, it should be fixed now.

**Comment by msaligane** [comment]
> And you may want to add a new line () before the pip installs in ReadMe section 2.2 to make sure each of them start with a new line. Sorry I wasn't aware of that when I modified that part.

You mean 2.4? can you make that change?

**Comment by QiruiZhang** [comment]
No just the 'pip install pyftdi' part in 2.2.

**Comment by msaligane** [comment]
Got it.

File: README.md [hunk 1]
```diff
@@ -96,6 +96,10 @@ To run the simulation, please edit your local model file in `common/platform_con
   }
 }
 ```
+# Tapeouts and testing setup
+
+Please refer to our testing setup in our tapeouts and testing setup [section](https://github.com/idea-fasoc/OpenFASOC/blob/7252190ebf6dd722e1c8e8ac48e2bf815e22b11e/tapeouts/MPWI/testsetup/README.md#section).
```
Comment (mithro) [inline README.md:101]: You should be able to use the following link;
```
[section](./tapeouts/MPWI/testsetup/README.md#section).
```

Comment (msaligane) [reply README.md:101]: thanks. This works better!

**Pull request merged**